### PR TITLE
Maker ethereum migration

### PR DESCRIPTION
**Pull request by Hosuke**
# Thank you for contributing to Spellbook!
Please refer to the top of the `readme` in the root of Spellbook to learn how to contribute to Spellbook on DuneSQL.

### CODE REVIEW

**Comment by Hosuke** [comment]
Sample maker_ethereum_accounting:
https://dune.com/queries/3064790

**Comment by jeff-dude** [comment]
@web3lyt do you want to help review the spell migration here?

**Comment by jeff-dude** [comment]
follow up:
@web3lyt is away at the moment, but mentioned another teammate may be able to help review

**Comment by jeff-dude** [comment]
we may need to prioritize this one this week @Hosuke, as we are getting closer to shutting down spark engine

fyi @aalan3 @couralex6 this one is higher priority to get done prior to shutdown

File: tests/maker/ethereum/maker_ethereum_accounting_test.sql
Comment (Hosuke) [inline tests/maker/ethereum/maker_ethereum_accounting_test.sql:1]: Based on the query result:
`abs_balance` (which is `sum(abs(value)) in 2022`) is 137785783606.1088.

The hard coded expected result in the test is 137785781602.9179.

They are having diff = 137785783606.1088-137785781602.9179 = 2003.1909
We can see if this difference/accuracy is acceptable.

https://dune.com/queries/3091415

**Comment by K-Lascar** [comment]
Thanks @Hosuke for noting the recent error: I've made a fix here: https://github.com/Hosuke/spellbook/pull/2
Please add it to your file!

**Comment by Hosuke** [comment]
> Thanks @Hosuke for noting the recent error: I've made a fix here: [Hosuke#2](https://github.com/Hosuke/spellbook/pull/2) Please add it to your file!

Thank you very much ser! The PR has been merged.

**Pull request merged**